scripts/templates/index.j2

```diff
@@ -0,0 +1,70 @@
+= Elastic Common Schema (ECS) Reference
+:doctype: book
+:ecs: ECS
+
+include::{asciidoc-dir}/../../shared/versions/stack/current.asciidoc[]
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+:ecs_github_repo_link: https://github.com/elastic/ecs/blob/{source_branch}
+:ecs_generated_version: {{ ecs_generated_version }}
```

Comment:
`:ecs_generated_version:` is defined here, but it's actually not being used.

The `{{ ecs_generated_version }}` fields (double curly braces) are getting replaced when Jinja2 renders the template. Using `{ ecs_generated_version }` (single curly brace) would remain for AsciiDoc to substitute the attribute when the docs get built. 

It's a bit confusing here since we've defined `ecs_generated_version` that's being passed to the Jinja2 templating and `:ecs_generated_version:` which is defined as an asciidoc attribute.

I don't have a preference if we use one way or the other, but if we're not actually using the asciidoc attribute I suggest we remove it entirely.

Reply:
Agree. I'd left it in case another `asciidoc` needs that at some time in the future - as you say, it's confusing, so  removed